modules/virt-creating-vm-web-page-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * virt/creating_vms_advanced/creating_vms_advanced_web/virt-creating-vms-from-web-images.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-vm-import-cli_{context}"]
+= Creating a VM from an image on a web page by using the command line
+
+You can create a virtual machine (VM) from an image on a web page by using the command line.
+
+When the virtual machine (VM) is created, the data volume with the image is imported into persistent storage.
```

Comment:
Since you added it in parentheses above, I don't think you need to expand it again

```suggestion
When the VM is created, the data volume with the image is imported into persistent storage.
```